doc/usr_51.jax

```diff
@@ -614,6 +571,9 @@ exists(":Cmd")		ユーザー定義コマンドが既にあるかどうかをチ
 	  enddef
 	endif
 <
+2回目のスクリプトのソース時に関数が削除されるのを防ぐため `vim9script` コマン
+ドに "noclear" を使用することを忘れないでください。
+
```

Comment:
ここもですが

```
Don't forget to use "noclear" with the `vim9script` command to avoid that the
function is deleted when the script is sourced a second time.
```

`to avoid that the function is deleted` ですが、2回目のsourceは *vim9-reload* にあるように再定義ではあるので、若干記述が間違いな気がしないでもないですね。

Reply:
お手間かけてすみません、いやまあそうですよね...